draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1295,30 +1298,31 @@ uCDN-A                         dCDN-B                         dCDN-C
 
    <t hangText="Name: total-objects-count">
    <vspace blankLines="1"/>
-   Description: Number of objects affected by the trigger, for example number of objects purged, invalidated or 
-   prepositioned as a result of trigger processing.  It is RECOMMENDED that dCDN maintains this attribtue as a cumulative
-   counter across all of its nodes, without de-duplicating same objects processed in multiple nodes. 
-   The primary purpose of this attribute is to help uCDN identify abnormal trigger processing results, that affected no
-   objects or unexpectedly low number of objects. 
-   dCDN MAY provide an updated object count as the trigger processing progresses in "active" trigger state,
+   Description: Total aggregate number of objects affected by the trigger, for example number of objects purged, 
+   invalidated or prepositioned as a result of trigger processing.  It is RECOMMENDED that dCDN maintains this 
+   attribute as a cumulative counter across all of its nodes, without de-duplicating same objects processed 
+   in multiple nodes. 
+   The primary purpose of this attribute is to help uCDN identify abnormal trigger processing results, e.g. purge or
```

Comment:
Nit:  Would read better if this said "a purge or preposition trigger", rather than "purge or preposition trigger"